scalafmt-benchmarks/src/resources/scala-js/OptimizerCore.scala

```diff
@@ -3086,7 +3086,7 @@ private[optimizer] abstract class OptimizerCore(
     }
   }
 
-  /** Tries and optimizes the remainings of a pattern match as if/elses.
+  /** Tries and optimizes the remains of a pattern match as if/elses.
```

Comment:
I'm ~60% confident here. (This is quite low for me)

Reply:
i would either restore the original, or change to `remaining cases`. the word "remains" means something else.